### PR TITLE
fix indentation of docstrings produced by renamed_arguments

### PR DESCRIPTION
**Pull request by dchudz**
Fixes https://github.com/HypothesisWorks/hypothesis-python/issues/1111 (just item 1, not the rest).

Before:

```
In [1]: from hypothesis.strategies import timedeltas

In [2]: ?timedeltas
Signature: timedeltas(min_value=datetime.timedelta(-999999999), max_value=datetime.timedelta(999999999, 86399, 999999), min_delta=None, max_delta=None)
Docstring:
A strategy for timedeltas between ``min_value`` and ``max_value``.

    Examples from this strategy shrink towards zero.



The following arguments have been renamed:

  * min_delta has been renamed to min_value
  * max_delta has been renamed to max_value

Use of the old names has been deprecated and will be removed
in a future version of Hypothesis.
File:      ~/hypothesis-python/src/hypothesis/strategies.py
Type:      function
```

After:

```
In [1]: from hypothesis.strategies import timedeltas

In [2]: ?timedeltas
Signature: timedeltas(min_value=datetime.timedelta(-999999999), max_value=datetime.timedelta(999999999, 86399, 999999), min_delta=None, max_delta=None)
Docstring:
A strategy for timedeltas between ``min_value`` and ``max_value``.

Examples from this strategy shrink towards zero.

The following arguments have been renamed:

  * min_delta has been renamed to min_value
  * max_delta has been renamed to max_value

Use of the old names has been deprecated and will be removed
in a future version of Hypothesis.
File:      ~/hypothesis-python/src/hypothesis/strategies.py
Type:      function
```

The difference is that `Examples from this strategy shrink towards zero.` is now in the right place.

We also have the corresponding improvement in the docs produced by `make documentation`.

### CODE REVIEW

File: tests/cover/test_renaming.py [hunk 1]
```diff
@@ -37,3 +39,15 @@ def test_using_old_args():
...
+
+
+@renamed_arguments(old_arg='new_arg')
+def g(new_arg=None, old_arg=None):
+    """Hi
+
+    Bye"""
+
+
+def test_docstring():
```
Comment (dchudz) [inline tests/cover/test_renaming.py]: Note: I confirmed that this test did in fact fail before the fix.

**Comment by dchudz** [comment]
(oops, forgot release file, coming up...)

**Comment by dchudz** [comment]
Sorry, I pushed a couple times before I was really finished and queued up some extra work for Travis. ☹️ 

This feature might be worthwhile to turn on: https://blog.travis-ci.com/2017-03-22-introducing-auto-cancellation

**Comment by Zac-HD** [comment]
No worries!  We've got autocancellation on for PRs, but it only cancels jobs that haven't *started* yet so there's usually still some lag in the system.

General review: very nice - core logic works, useful test, and I like that you left a note pointing back to the unfixed issues.  Once the build passes I'll merge!

**Comment by dchudz** [comment]
> (note re: trailing whitespace - your editor should have a setting to strip it automatically when saving)

Ah, sorry. Done now. (I had that already where I edit Python code, but have got it on in Sublime now too. Thanks.)

Thanks for showing me `inspect.cleandoc`. I didn't know anything about how docstrings end up looking okay before. 😄

**Pull request merged**